package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "synthetic-monitoring-app",
-  "version": "1.9.36",
+  "version": "1.9.40",
```

Comment:
I think you could go `1.10.0` if you wanted to count this as a feature instead of a patch